cli/src/commands/lint.ts

```diff
@@ -12,7 +12,8 @@ export interface LintConfig {
 
 const lintConfig: LintConfig = {
   rules: {
-    "no-omittable-fields-within-response-bodies": "warn"
+    "no-omittable-fields-within-response-bodies": "warn",
+    "no-trailing-forward-slash": "warn"
```

Comment:
This is the default configuration used by the `lint` command. It can be overriden via command-line arguments since https://github.com/airtasker/spot/pull/2153